Automattic/jetpack
# Jetpack Debug: Update Sync related debugging info

This PR updates/removes some of the public facing debugging info we display to our end users when Jetpack Sync is experiencing issues.

Ideally our end users don't need to know anything about Jetpack Sync, since it might be powering most of our Jetpack products and features, but is not considered a public-facing tool itself.

## Proposed changes:

* Full Sync tests have been removed, since Full Sync is now considered deprecated in favour of Sync Checksums.
* Triggering a Full Sync when Sync is experiencing Data Loss has been removed. We already have self-healing mechanisms in place, on WPCOM, when a site is experiencing Data Loss. Assuming the issue persists, we now advise our end users to contact Support instead of attempting to trigger a Full Sync. This also enabled us to remove the following files: `jetpack-debugger-site-health.css`, `jetpack-debugger-site-health.js`
* The informational tests, when Sync is experiencing delay, have been removed as well.

### Other information:


## Jetpack product discussion


pf5801-sb-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:





* Make sure to compare the Site Health tool and Jetpack Debugger page in wp-admin (`wp-admin/admin.php?page=jetpack-debugger`) with/without the PR
* Update the Sync Health to `out_of_sync`
* Visit the Site Health tool and confirm the message is updated as per pf5801-sb-p2#comment-211
<img width="848" alt="Screenshot 2024-01-16 at 11 10 32" src="https://github.com/Automattic/jetpack/assets/1758399/c39bf094-afdf-4c49-8e5b-40753f447095">

* Visit the Jetpack Debugger page in wp-admin and confirm the message is updated as per pf5801-sb-p2#comment-211
<img width="1082" alt="Screenshot 2024-01-16 at 11 11 05" src="https://github.com/Automattic/jetpack/assets/1758399/31590a20-82e9-4e73-a491-ecfab355e9ef">

* Update the Sync Health to `in_sync`
* Visit the Site Health tool / Debugger page and confirm no Sync related messages show up
* Disable Sync 
* Visit the Site Health tool and confirm the message is updated as per pf5801-sb-p2#comment-213
<img width="837" alt="Screenshot 2024-01-16 at 11 14 08" src="https://github.com/Automattic/jetpack/assets/1758399/bf0a5db0-f2c0-40cb-bc41-99717139eb74">

* Visit the Jetpack Debugger page in wp-admin and confirm the message is updated as per pf5801-sb-p2#comment-213
<img width="1071" alt="Screenshot 2024-01-16 at 11 13 32" src="https://github.com/Automattic/jetpack/assets/1758399/aeae3f7b-4c90-4c54-9f53-e639b27e053c">
